tools/test.py

```diff
@@ -1273,6 +1276,24 @@ def ProcessOptions(options):
   VERBOSE = options.verbose
   options.arch = options.arch.split(',')
   options.mode = options.mode.split(',')
+  options.run = options.run.split(',')
+  if options.run == [""]:
+    options.run = None
+  elif len(options.run) != 2:
+    print "The run argument must be two comma-separated integers."
+    return False
+  else:
+    try:
+      options.run = map(int, options.run)
+    except ValueError:
+      print "Could not parse the integers from the run argument."
+      return False
+    if options.run[0] < 0 or options.run[1] < 0:
+      print "The run argument cannot have negative integers."
+      return False
+    if options.run[0] >= options.run[1]:
+      print "The test to run (n) must be smaller than the modulo (m)."
```

Comment:
Sorry. I forgot to mention about this last time. The skip value can be lesser than the start value right? For example, start from 5th test skip two tests, in that case it will be `-r=5,2`. Please correct me if I am missing something